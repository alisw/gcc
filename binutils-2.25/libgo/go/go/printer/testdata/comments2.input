../../../../../../libgo/go/go/printer/testdata/comments2.input