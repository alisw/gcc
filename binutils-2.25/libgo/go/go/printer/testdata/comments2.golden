../../../../../../libgo/go/go/printer/testdata/comments2.golden